config/common/common.views.xml

```diff
@@ -2598,34 +1910,32 @@
 					<cell type="label" labelfor="name"/>
 					<cell type="field" id="name" name="name" uitype="text"/>
 					<cell type="label" labelfor="cogType"/>
-					<cell type="field" id="cogType" name="cogType" uitype="combobox" picklist="COGTypes"/>
-					<cell type="label" labelfor="parentCojo"/>
-					<cell type="field" id="parentCojo" name="parentCojo" uitype="querycbx" isrequired="false"/>
+					<cell type="field" id="cogType" name="cogType" uitype="combobox"/>
+					<cell type="label" label="Parent COG"/>
+					<cell type="field" id="parentCojo" name="parentCojo" uitype="querycbx"/>
 				</row>
 				<row>
-					<cell type="subview" id="cojo" name="cojo" uitype="subview" isrequired="false" defaulttype="table" colspan="13"/>
+					<cell type="subview" id="cojo" name="children" uitype="subview" defaulttype="table" colspan="13"/>
 				</row>
 			</rows>
 		</viewdef>
-
 		<viewdef name="CollectionObjectGroupSub" class="edu.ku.brc.specify.datamodel.CollectionObjectGroup" type="form" gettable="edu.ku.brc.af.ui.forms.DataGetterForObj" settable="edu.ku.brc.af.ui.forms.DataSetterForObj">
 			<desc>Collection Object Group Subform</desc>
 			<enableRules/>
-			<columnDef>50px,2px,150px,5px,50px,2px,150px,5px,50px,2px,200px,2px,50px,p:g</columnDef>
+			<columnDef>50px,2px,150px,5px,50px,2px,150px,5px,50px,2px,p:g</columnDef>
 			<rowDef auto="true" cell="p" sep="2px"/>
 			<rows>
 				<row>
 					<cell type="label" labelfor="name"/>
 					<cell type="field" id="name" name="name" uitype="text"/>
 					<cell type="label" labelfor="cogType"/>
 					<cell type="field" id="cogType" name="cogType" uitype="combobox" picklist="COGTypes"/>
-					<cell type="label" labelfor="parentCojo"/>
-					<cell type="field" id="parentCojo" name="parentCojo" uitype="querycbx" isrequired="false"/>
-					<cell type="subview" id="cojo" name="cojo" uitype="subview" isrequired="false" defaulttype="table" initialize="btn=true; align=center" colspan="13"/>
+					<!--<cell type="label" labelfor="parentCojo"/>
+					<cell type="field" id="parentCojo" name="parentCojo" uitype="querycbx" isrequired="false"/> -->
+					<cell type="subview" id="cojo" name="childCojos" uitype="subview" initialize="btn=true"/>
```

Comment:
`childCojos` would have to be changed to `children`

Reply:
Fixed in [`a2108ab` (#5367)](https://github.com/specify/specify7/pull/5367/commits/a2108ab50e85e7053eac5ed6d6db865c2a3fa302)